README.md

```diff
@@ -3,5 +3,28 @@ Repositorio para la asignatura Infraestructura Virtual de Antonio Rodríguez Rod
 
 ## Problema a resolver:
 
-Soy un estudiante que tiene el problema de ser muy exquisito con la comida y quiero saber cuando se va a ofrecer comida que se adapte a mis preferencias, ya que pierdo mucho tiempo teniendo que entrar en la web de comedores universitarios y ver entre todo lo disponible si me gusta algo. 
-Del mismo modo, me gustaría saber la calidad alimenticia de los platos que hay disponibles cada día: macronutrientes, calorías, etc. Ya que tengo como objetivo personal, con mi entrenamiento, subir de peso ganando masa muscular, con lo que debería tener alguna forma de adecuar mi dieta a dicho objetivo.
\ No newline at end of file
+Soy un estudiante que tiene el problema de ser muy exquisito con la comida y quiero saber cuando se va a ofrecer comida que se adapte a mis preferencias, ya que pierdo mucho tiempo teniendo que entrar en la web de comedores universitarios y ver entre todo lo disponible si me gusta algo. Del mismo modo, me gustaría saber la calidad alimenticia de los platos que hay disponibles cada día: macronutrientes, calorías, etc. Ya que tengo como objetivo personal, con mi entrenamiento, subir de peso ganando masa muscular, con lo que debería tener alguna forma de adecuar mi dieta a dicho objetivo.
+
+# Resolución del problema
+
+## Descripción
+Este proyecto es un sistema automatizado que ayuda a los estudiantes a encontrar comidas que se adapten a sus preferencias y objetivos nutricionales. El sistema extrae los menús diarios de diferentes comedores disponibles, filtra los platos según las preferencias del usuario y proporciona un análisis nutricional de los platos.
+El estudiante evita perder tiempo revisando los menús manualmente y puede seguir un plan nutricional adecuado a sus necesidades de entrenamiento.
+
+## [Journey](https://github.com/antoniorr02/MenuConsulter/blob/Objetivo-1/docs/journeys.md)
+- Un estudiante universitario, es muy selectivo con la comida y quiere asegurarse de que cada día haya opciones que le gusten en el comedor. Tiene poco tiempo y no le gusta revisar los menús manualmente.
+- Una estudiante que entrena para ganar masa muscular, quiere asegurarse de que las comidas que toma contribuyen a su objetivo de consumo de proteínas y calorías.
+
+## [Historias de Usuario](https://github.com/antoniorr02/MenuConsulter/blob/Objetivo-1/docs/historias_usuario.md)
+- **[HU1]:** Ojalá me avisaran cuando haya platos que se ajusten a mis preferencias alimenticias.
+- **[HU2]:** Me gustaría poder ver un análisis de macronutrientes de cada plato para ajustar mi dieta según mis objetivos de entrenamiento.
+- **[HU3]:** Ojalá pudiera filtrar los menús disponibles según mis objetivos nutricionales.
+
+## [Hitos del Proyecto](https://github.com/antoniorr02/MenuConsulter/blob/Objetivo-1/docs/milestones.md)
+El desarrollo se divide en varios hitos para asegurar que se cumplan los objetivos de manera eficiente:
+
+- **Hito 0:** Entendimiento de la resolución del problema.
```

Comment:
¿Crees que estos hitos están formulados como productos mínimamente viables? ¿Dónde está el producto? ¿Dónde la viabilidad?

Reply:
He actualizado el documento de los milestones con la información que me solicita, le invito a revisarlo ahora a ver si se adecua a los requisitos de la práctica.

Reply:
¿Crees que "entendimiento" describe un producto?

Reply:
Tal y como se habló en clase, se propuso que el hito 0 fuera la comprensión de cómo se iba a obtener la solución del problema. Del mismo modo, el producto se describe dentro del documento milestones.md que se enlaza en el readme, donde se desarrolla cada hito de una forma más detallada.

Reply:
Un hito nunca puede ser "entendimiento" o "comprensión". Tiene que ser un producto, e hicimos mucho énfasis en clase de que así fuera. Si no entiendes como formular un producto, es imposible que lo entregues al cliente o compruebes su viabilidad, y en ingeniería es imprescindible poder trabajar con productos y pruebas objetivas.

Reply:
Sería correcto definirlo como: Toma de decisiones para la resolución del problema.

Ya que al fin y al cabo lo que se busca con este hito es decidir qué tecnologías y con qué metodología se va a comenzar el desarrollo.

Reply:
¿Toma de decisiones es un producto? Como pone en [el guión](http://jj.github.io/IV/documentos/proyecto/1.Planificacion)

> Finalmente, los diferentes productos que se van a entregar a estos clientes (o al equipo de desarrollo antes de desarrollar los productos que se vayan a entregar al cliente) se tendrán que describir en una serie de hitos o milestones, en cada uno de los cuales se entregará un producto mínimamente viable. El desarrollo ágil se basa en un método iterativo en el que se mejora un producto, siempre con la aprobación del usuario. Por lo tanto, tendrá que haber al menos un par de milestones que refleje los diferentes productos que se van a entregar y que vayan agrupando las historias de usuario y los diferentes issues que se saquen de las historias de usuario.

Reply:
Bueno he cambiado el título del hito por Modelización del problema, sin embargo, me da la sensación de que más que un problema de entendimiento de la metodología MVP es más un mal entendimiento a nivel lingüistico. En cualquier caso, creo que queda claro que producto se intenta abordar en este milestone.

Reply:
¿Modelización es un producto? En todo caso, cuando termines solicita explícitamente revisión, por favor.

Reply:
Pensaba que todos los comentarios eran revisiones para mejorar y superar el objetivo 1, en cualquier caso ante la negativa de cualquier otro posible nombre le he puesto a este milestone modelo del problema, tal y como tiene marcado Giorgio en su repositorio (el único que ha superado el objetivo 1 por ahora).

Con todo ello creo que sería conveniente ya la revisión de mi objetivo, gracias de antemano.

Reply:
¿Te das cuenta de que el hecho de que digas "lo he hecho como Giorgio" implica que no has entendido lo que se pide en el objetivo, que has puesto simplemente lo que has visto y que por tanto me vería obligado a solicitarte cambios al mismo?
El trabajo es individual, y lo que se entrega también es individual. Si no se entiende lo que se busca del milestone 0, es imposible que puedas hacerlo. El trabajo en ingeniería se basa en entender qué es lo que hay que hacer a continuación y si lo que hago está bien. "Lo que ha escrito Giorgio", ¿crees que te ayudará a entender de forma autónoma cuando has hecho "un modelo del problema" y por tanto has terminado el milestone?
Ese milestone, precisamente, es uno en el que otra persona tiene que desarrollar para ti. Si no  entiendes qué es lo que tiene que entregar esa otra persona, sino que lo único que sabes es "lo que ha escrito Giorgio", ¿crees que serás capaz de aprobar ese PR validando por tanto el producto entregado?
En todo caso, te agradezco la sinceridad. Te rogaría por favor que revisaras el guión de este objetivo, para ver qué es realmente un milestone/PMV y describieras con tus propias palabras qué es lo que aceptarás como válido cuando te lo entreguen.

Reply:
Me he leído la docuementación de la práctica completa unas 4 veces (la cuál se hace bastante liosa al ser tan extensa),  llevo 3 días proponiendo sinónimos para algo tan subjetivo como un producto, muchisimas empresas no sólo de ámbito informático, pueden ser de ámbito financiero, contable, abogacía, asesoria, etc; ofrecen como productos soluciones o el planteamiento de la resolución de los mismos, tal y cómo lo indiqué en el primer PR, todo esto es un producto y ni tan siquiera requiere de ser tangible. Con lo que repito creo que el objetivo de la práctica lo tengo más que asimilado y se trata de lo que usted y yo consideramos como producto, o la palabra clave que cada uno consideramos que puede describirlo a nivel meramente lingüistico.

En cualquier caso he vuelto a revisar la documentación relativa a este apartado y he introducido ciertas mejoras dentro del archivo correspondiente. Con todo ello quedo a la espera de saber si hay alguna otra pega para dar el objetivo como superado.

Reply:
> Me he leído la docuementación de la práctica completa unas 4 veces (la cuál se hace bastante liosa al ser tan extensa), llevo 3 días proponiendo sinónimos para algo tan subjetivo como un producto, muchisimas empresas no sólo de ámbito informático, pueden ser de ámbito financiero, contable, abogacía, asesoria, etc; ofrecen como productos soluciones o el planteamiento de la resolución de los mismos, tal y cómo lo indiqué en el primer PR, todo esto es un producto y ni tan siquiera requiere de ser tangible. Con lo que repito creo que el objetivo de la práctica lo tengo más que asimilado y se trata de lo que usted y yo consideramos como producto, o la palabra clave que cada uno consideramos que puede describirlo a nivel meramente lingüistico.
> 
> En cualquier caso he vuelto a revisar la documentación relativa a este apartado y he introducido ciertas mejoras dentro del archivo correspondiente. Con todo ello quedo a la espera de saber si hay alguna otra pega para dar el objetivo como superado.

Habrás hecho todo eso, pero no has contestado a esta pregunta: ¿crees que serás capaz de aprobar ese PR validando por tanto el producto entregado? Como no puede ser de otro modo, leeré lo que escribas. Pero lo único que se busca aquí es que cuando llegues al objetivo 2 y "valides" el desarrollo que la persona que toque en tu repo haga, no tenga que haber 10 o 12 rondas adicionales de validación por mi persona para que realmente haya un producto que puedas usar en los objetivos sucesivos.

Tarde o temprano tendréis que entender que lo que se entrega aquí no se trata de un "ejercicio" o de "entregar lo que ha escrito no sé quién". Se trata de que tengáis herramientas para proceder, de forma autónoma, con el desarrollo de un producto informático.